pennylane/utils.py

```diff
@@ -239,6 +240,10 @@ def inv(operation_list):
     If the inversion happens inside a QNode, the operations are removed and requeued
     in the reversed order for proper inversion.
 
+    .. warning::
+        Use of :func:`~.qml.inv()` is deprecated and should be replaced with
+        :func:`~.qml.adjoint()`.
```

Comment:
Good idea adding it to the docstring! However, if you check the rendered docstring: https://pennylane--1325.org.readthedocs.build/en/1325/code/api/pennylane.inv.html, the two functions are not clickable.

This is because `qml` is only a convention; the actual package name is `pennylane`. However, we can actually remove the `.qml` prefix altogether -- because there is only one `inv` and `adjoint` function in the code-base, Sphinx will be able to find it automatically!

```suggestion
    .. warning::
        Use of :func:`~.inv()` is deprecated and should be replaced with
        :func:`~.adjoint()`.
```